LayoutTests/inspector/css/getComputedPrimaryFontForNode.html

```diff
@@ -220,6 +287,13 @@
     <div id="webfont-cjk-test-2" style="font-family: SomethingElse, WebFontWithCJKAndLatin, sans-serif;">&#x00A5Yen</div>
 
     <div id="variable-font-test-1" style="font-family: VariableFont;">Variable Font Test 1</div>
+
+    <div id="synthetic-weight-test-1" style="font-family: AhemNormal; font-weight: normal;">Synthetic Weight Test 1</div>
+    <div id="synthetic-weight-test-2" style="font-family: AhemNormal; font-weight: bold;">Synthetic Weight Test 2</div>
```

Comment:
I think more descriptive names here would be great.  Maybe something like `synthetic-weight-from-normal` and `synthetic-weight-from-bold`?

ditto below